docs/docs/docs/01-core/strapi/commands/plugin/05-link-watch.md

```diff
@@ -0,0 +1,33 @@
+---
+title: plugin:link-watch
```

Comment:
Since it encompasses the `plugin:watch` command but adds the linking to it, would it make sense to call it `plugin:watch:link` ?

Reply:
Yeah I think that's probably sensible, it follows the same pattern as `plugin:something`